webapp-django/crashstats/crashstats/jinja2/crashstats/report_index.html

```diff
@@ -691,44 +691,46 @@ <h2>{{ report.product }} {{ report.version }} Crash Report [@ {{ report.signatur
               </tbody>
             </table>
 
-            {% if (request.user.has_perm('crashstats.view_pii') or your_crash) and report.breadcrumbs %}
-              <div id="breadcrumbs">
-                <h2>Breadcrumbs</h2>
-                <p>
-                  {{ protected_warning(your_crash) }} -
-                  {% if report.breadcrumbs|length > 10 %}
-                    Latest 10 breadcrumbs (out of {{ report.breadcrumbs|length }})
-                  {% else %}
-                    All {{ report.breadcrumbs|length }} breadcrumbs
+            {% if java_exception_stacks %}
```

Comment:
I moved breadcrumbs to below showing stack data.

I modeled showing stacks here to be like showing crashing thread stacks.

I think it's kind of gross looking, but Socorro doesn't have good primitives for this (imo) and I didn't want to figure that out in addition to everything else. I think it behooves us to do a ui/ux refresh and deal with it in that.